Week1/homework/app.js

```diff
@@ -1,11 +1,129 @@
-'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  'use strict';
+
+  //  1.1
+
+  const myBooks = [
+    'notes_from_underground',
+    'brave_new_world',
+    'nineteen_eighty_four',
+    'animal_farm',
+    'the_little_prince',
+    'the_alchemist',
+    'essays_montaigne',
+    'blindness',
+    'a_confession',
+    'sherlock_holmes',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  // 1.2
+  console.log(myBooks);
+
+  //  1.3
+
+  function makeList() {
+    const myBooks = [
+      'notes_from_underground',
+      'brave_new_world',
+      'nineteen_eighty_four',
+      'animal_farm',
+      'the_little_prince',
+      'the_alchemist',
+      'essays_montaigne',
+      'blindness',
+      'a_confession',
+      'sherlock_holmes',
+    ];
+
+    const listDiv = document.createElement('div');
+    document.getElementsByTagName('body')[0].appendChild(listDiv);
+
+    const ulTag = document.createElement('ul');
+    listDiv.appendChild(ulTag);
+
+    for (let i = 0; i < myBooks.length; i++) {
+      const liTag = document.createElement('li');
+      liTag.innerHTML = myBooks[i];
+      ulTag.appendChild(liTag);
+    }
+  }
+
+  // makeList(); 
+  //I canceled function calling to avoid showing on page
```

Comment:
You should update your function to work with the details of your book (now you have the same functionality twice in this file), or maybe eve n better: call the `makeList()` function from the body of your `infoBookList()` function.

Reply:
I changed my function system and canceled this part